v3/lints/cabf_br/lint_underscore_not_permissible_in_dnsname.go

```diff
@@ -0,0 +1,54 @@
+/*
+ * ZLint Copyright 2021 Regents of the University of Michigan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package cabf_br
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/zmap/zlint/v3/util"
+
+	"github.com/zmap/zcrypto/x509"
+	"github.com/zmap/zlint/v3/lint"
+)
+
+func init() {
+	lint.RegisterLint(&lint.Lint{
+		Name:          "e_underscore_not_permissible_in_dnsname",
+		Description:   "DNSNames MUST NOT contain underscore characters",
+		Citation:      "BR 7.1.4.2.1",
+		Source:        lint.CABFBaselineRequirements,
+		EffectiveDate: util.BALLOT_SC_12_Ineffective,
```

Comment:
I realize we're all over the board with dates, so this is very much a "nit" (and already the suggestion elsewhere to rename this, which I suppose this would replace), but wdyt about

`CABFBRs_1_6_2_UnderscoreSunsetDate`

It's very long, so it may be a terrible name though.

Reply:
The interesting thing about this repo and is that how often "breaking the rules" is actually  the preferred option.

`LintThisButOnlyWhenThatAndOnlyOnTheseDays`

This is an awful name, but the alternative would be vague (at best) and most likely misleading.